docs/source/contributing/jupyter_style.md

```diff
@@ -186,14 +186,23 @@ The syntax is as follows:
 
 The date should correspond to the latest update/execution date, at least roughly (it's not a problem if the date is a few days off due to the review process before merging the PR). This will allow users to see which notebooks have been updated lately and will help the PyMC team make sure no notebook is left outdated for too long.
 
-The {ref}`MyST target <myst:syntax/targets>`
+The {ref}`MyST target <myst:syntax/targets>` (the `(notebook_name)=` bit)
 is important to ease referencing and linking notebooks between each other.
+It must be notebook specific, for example its file name.
 
 Tags can be anything, but we ask you to try to use [existing tags](https://github.com/pymc-devs/pymc/wiki/Categories-and-Tags-for-PyMC-Examples)
 to avoid the tag list from getting too long.
 
-Each notebook should have a single category indicating the level of the notebook.
-Choose a category from [existing categories](https://github.com/pymc-devs/pymc/wiki/Categories-and-Tags-for-PyMC-Examples#categories).
+Each notebook should have a one or two categories indicating:
```

Comment:
just make it two so its not ambiguous

Reply:
The problem is that most notebooks are a mix of different types, so forcing the two categories already is a huge effort. My idea was to use that for new notebooks, for notebooks that already match a type and for notebooks we want to split/combine/restructure

Reply:
My suggestion is force two moving forward and we can fix the older ones over time. This way at least at some point in the future all notebooks will have both labels

Reply:
sounds good, I have updated the proposal template in https://github.com/pymc-devs/pymc-examples/pull/281. I'd still keep it like this here for now because >95% of the changes that we have to do (and 99.9% of the changes done by newcomers) are updating existent notebooks, not writing new ones